website/src/common/Footer.jsx

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+
+import {
+  AiFillGithub,
+  AiFillSlackCircle,
+  AiFillTwitterCircle,
+} from "react-icons/ai";
+import { BsStackOverflow } from "react-icons/bs";
+import { Link } from "react-router-dom";
+
+const Footer = () => {
+  const socials = [
+    { url: "https://github.com/vitessio/arewefastyet", icon: AiFillGithub },
+    { url: "https://vitess.io/slack", icon: AiFillSlackCircle },
+    { url: "https://twitter.com/vitessio", icon: AiFillTwitterCircle },
+    { url: "https://stackoverflow.com/search?q=vitess", icon: BsStackOverflow },
+  ];
+
+  const items = [
+    {
+      title: "Benchmarks",
+      links: [
+        { title: "Daily", to: "/daily" },
+        { title: "Compare", to: "/compare" },
+        { title: "Micro", to: "/micro" },
+        { title: "Macro", to: "/macro" },
+        { title: "PR", to: "/pr" },
+      ],
+    },
+    {
+      title: "Vitess",
+      links: [
+        { title: "Docs", to: "https://vitess.io/docs" },
+        { title: "Blog", to: "https://vitess.io/blog" },
+        { title: "Community", to: "https://vitess.io/community" },
+      ],
+    },
+    {
+      title: "arewefastyet",
+      links: [{ title: "github", to: "https://www.github.com/vitessio/arewefastyet" }],
```

Comment:
Let's change the title here to `GitHub`
```suggestion
      links: [{ title: "GitHub", to: "https://www.github.com/vitessio/arewefastyet" }],
```